pyrr/geometric_tests.py

```diff
@@ -316,17 +316,17 @@ def point_height_above_plane(point, plane):
         negative if the point is behind the plane.
     """
     """
-    http://www.vitutor.com/geometry/distance/point_plane.html
-    d(P) = (AX + BY + CZ + D) / sqrt(A^2 + B^2 + C^2)
-
-    Normal is unit length, so it's length is 1.0.
-    Therefore, we can ignore the division all together.
-    Just perform Pn . [XYZ1]
+    Because we store normalised normal, we can simply
```

Comment:
In my oppinion this is a good Idea, but I think the create Functions should make sure that this is the case

Reply:
It should really have a normalise call in there, there's no guarantee in the type that its normalised. You could pass in your own hand-made data.

Reply:
If you think people will use non normalized planes, then `plane.normal` and `plane.distance` should be divided by `np.linalg.norm(pl[:3])`. Personally I would be fine in saying the function wants a plane that is in accordance to the Hessian normal form